src/libraries/Microsoft.Extensions.Http/src/Logging/LoggingScopeHttpMessageHandler.cs

```diff
@@ -43,6 +55,7 @@ public LoggingScopeHttpMessageHandler(ILogger logger, HttpClientFactoryOptions o
             _options = options;
         }
 
+        /// <inheritdoc />
```

Comment:
ditto.

Reply:
ditto


If not sufficient please let me know and I'll be glad to copy the inherited documentation forward explicitly.